test/rendition.postprocess.test.js

```diff
@@ -43,7 +43,7 @@ mockRequire("@adobe/asset-compute-image-processing", {
             } else if (instructions.fmt === 'tiff') {
                 await fs.copyFile('test/files/generatedFileSmall.tiff',outfile);
             } else {
-                throw new Error('unknown error');
+                throw new GenericError('batch worker renditions postprocess applies only for all renditions');
```

Comment:
if you look at asset-compute-image-processing, we throw actual Errors (not GenericErrors): https://github.com/adobe/asset-compute-image-processing/blob/master/lib/image.js#L164 and that's why I had it throw an Error.

this error is then caught in worker.js and turned into a `GenericError`

Reply:
thanks, had changed it for testing GenericError else where. will revert.